docs/operational-metrics.md

```diff
@@ -0,0 +1,39 @@
+
+> Note: this file was automatically generated, to update execute "make docs"  
+  
+# flowlogs-pipeline Operational Metrics  
+  
+Each table below provides documentation for an exported flowlogs-pipeline operational metric. 
```

Comment:
@ronensc It might ... if this is ok with you, I would like to keep this as-is for now. This is aligned with the other metrics (network metrics) documentation format ... I agree that we can change this / improve etc ... I was thinking to add a description field with details etc. I hope that the RH team will be able to put that in front of tech-writer so get professional review :-)